docs/running-on-coclobas.md

```diff
@@ -20,12 +20,11 @@ of them.
 You'll need the GCloud command line utility, installed and initialized
 [from here](https://cloud.google.com/sdk/downloads#interactive).
 
-Optionally set your preferred zone and cluster sizes before you get started:
+Optionally set your preferred zone before you get started:
 
 ```shell
-# disco.sh defaults
+# disco.sh default
 export GCLOUD_ZONE="us-east1-c"
```

Comment:
the gcloud-zone should also be on the the server-host right?
(it's necessary to create the host and also used inside the configuration generated by `disco.sh` on the newly created node).